pom.xml

```diff
@@ -35,7 +35,8 @@
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <jetty.version>9.3.8.v20160314</jetty.version>
     <appengine.api.version>1.9.40</appengine.api.version>
-    <docker.label.prefix>async-</docker.label.prefix>
+    <docker.tag>${jetty.version}</docker.tag>
```

Comment:
We're trying to be consistent with other language runtimes, and they seem to be adopting the timestamp `YYYY-MM-DD_HH_MM` as the most specific docker version tag.